flink-runtime/src/test/java/org/apache/flink/runtime/security/token/hadoop/HadoopFSDelegationTokenProviderITCase.java

```diff
@@ -129,32 +127,30 @@ protected void obtainDelegationTokens(
                     }
                 };
 
-        assertEquals(
-                Optional.of(1L),
-                provider.getTokenRenewalInterval(constantClock, Collections.emptySet()));
+        assertThat(provider.getTokenRenewalInterval(constantClock, Collections.emptySet()))
+                .isEqualTo(Optional.of(1L));
```

Comment:
assertThat(provider.getTokenRenewalInterval(constantClock, Collections.emptySet()).get())
                .isOne();

Reply:
Assertj supports Optional. How about using hasValue() like this?
```
assertThat(provider.getTokenRenewalInterval(constantClock, Collections.emptySet()))
                .hasValue(1L);
```